helix-core/src/test/java/org/apache/helix/integration/task/TestWorkflowTermination.java

```diff
@@ -157,7 +157,7 @@ public void testWorkflowPausedTimeout() throws Exception {
     Thread.sleep(100);
 
     // Pause the queue
-    _driver.waitToStop(workflowName, 10000L);
+    _driver.waitToStop(workflowName, 20000L);
 
```

Comment:
can we use verifier?

Reply:
As we talked offline, this is not a boolean returning method and so can't use verifier.